### DISCUSSION TITLE
Simple YAML file crashes with "Invalid indent query"

### DISCUSSION THREAD

**Discussion by notmgsk** [helix-editor/helix#8297] [Troubleshooting]
### Summary

When editing a very simple file
```
foo:
- bar:
```
helix crashes if I place my cursor at the end of `bar:` and type `o`.

This is the error message:
```
thread 'main' panicked at 'Invalid indent query: Unknown predicate ("not-one-line?")', helix-core/src/indent.rs:392:21
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

### Reproduction Steps

1. Create a YAML file with contents
```yaml
foo:
- bar:
```

2. Put cursor at the end of `bar:`
3. Type `o` (when in `NOR` mode)

I expected this to happen: newline below `bar:`.

Instead, this happened:
```
thread 'main' panicked at 'Invalid indent query: Unknown predicate ("not-one-line?")', helix-core/src/indent.rs:392:21
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

```
2023-09-15T15:58:18.628 helix_view::editor [ERROR] Failed to initialize the LSP for `source.yaml` { cannot find binary path }
```

</details>

### Platform

Linux

### Terminal Emulator

gnome-terminal

### Helix Version

helix 23.05 (7f5940be)

**Comment by pascalkuthe** [comment]
you are using the latest stable release with runtime files copied from the master branch.

Runtime files are not compatible with different versions
